security-response.md

```diff
@@ -0,0 +1,102 @@
+# Security Response Guidelines
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable to
+publicly known exploits.
+
+The OpenTelemetry Technical Committee (OTel TC) and relevant repo maintainers,
+supported by tooling provided by the SIG-Security, are responsible for
+responding to the incident organizing the entire response including internal
+communication and external disclosure.
+
+## Supported Versions
+
+The OTel project provides community support only for the last overall minor
```

Comment:
what happens for problems affecting only older versions? are users required to migrate to the latest?
is this a current policy, or is it new? if it's new, we should bring this to the TC/maintainers - @jpkrohling

Reply:
This is the existing policy from the security page in our community GitHub